File: JAVA/petshop.java
public class petshop{
    private int id,harga,stok;
    private String nama;

    public petshop(){
        this.id=0;
        this.harga=0;
        this.stok=0;
        this.nama="";
    }
    
    public petshop(int id,int harga,int stok,String nama){
        this.id= id;
        this.harga=harga;
        this.stok=stok;
        this.nama=nama;
    }

    
    public int getid(){
        return this.id;
    }
    public void setid(int id){
        this.id = id;
    }
    
    public int getharga(){
        return this.harga;
    }
    public void setharga(int harga){
        this.harga = harga;
    }
    
    public int getstok(){
        return this.stok;
    }
    public void setstok(int stok){
        this.stok = stok;
    }

    public String getNama(){
        return this.nama;
    }
    public void setNama(String nama){
        this.nama = nama;
    }
    
}

File: PHP/petshop.php
<?php
class petshop{
    private $id=0;
    private $harga=0;
    private $stok=0;
    private $nama='';
    private $foto='';
    public function __construct($id=0,$harga=0,$stok=0,$nama='',$foto=''){
        $this->id= $id;
        $this->harga= $harga;
        $this->stok= $stok;
        $this->nama= $nama;
        $this->foto= $foto;
    }
    
    public function setid($id){
        $this->id= $id;
    }
    public function getid(){
        return $this->id;
    }
    
    public function setharga($harga){
        $this->harga= $harga;
    }
    public function getharga(){
        return $this->harga;
    }
    
    public function setstok($stok){
        $this->stok= $stok;
    }
    public function getstok(){
        return $this->stok;
    }

    public function setnama($nama){
        $this->nama= $nama;
    }
    public function getnama(){
        return $this->nama;
    }
    
    public function setfoto($foto){
        $this->foto= $foto;
    }
    public function getfoto(){
        return $this->foto;
    }
    
}
?>